code/game/machinery/status_display.dm

```diff
@@ -32,6 +36,51 @@
 	var/index1 // display index for scrolling messages or 0 if non-scrolling
 	var/index2
 
+/obj/item/wallframe/status_display
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon_state = "unanchoredstatusdisplay"
+	custom_materials = list(/datum/material/iron=14000, /datum/material/glass=8000)
+	result_path = /obj/machinery/status_display
+
+//Copy pasted decon paths from newscaster.dm
+/obj/machinery/status_display/wrench_act_secondary(mob/living/user, obj/item/tool)
+	. = ..()
+	user.balloon_alert(user, "[anchored ? "un" : ""]securing...")
+	tool.play_tool_sound(src)
+	if(tool.use_tool(src, user, 6 SECONDS))
+		playsound(loc, 'sound/items/deconstruct.ogg', 50, TRUE)
+		user.balloon_alert(user, "[anchored ? "un" : ""]secured...")
+		deconstruct()
+		return TRUE
+
+/obj/machinery/status_display/welder_act(mob/living/user, obj/item/tool)
+	if(machine_stat & BROKEN)
+		if(!tool.tool_start_check(user, amount=0))
+			return
+		user.visible_message(span_notice("[user] is repairing [src]."), \
+							span_notice("You begin repairing [src]..."), \
+							span_hear("You hear welding."))
+		if(tool.use_tool(src, user, 4 SECONDS, volume=50))
+			if(!(machine_stat & BROKEN))
+				to_chat(user, span_notice("[src] does not need repairs."))
+				return
+			to_chat(user, span_notice("You repair [src]."))
+			atom_integrity = max_integrity
+			set_machine_stat(machine_stat & ~BROKEN)
+			update_appearance()
+	else
+		return ..()
+
+/obj/machinery/status_display/deconstruct(disassembled = TRUE)
+	if( !flags_1 && !NODECONSTRUCT_1 && !disassembled )
+		new /obj/item/stack/sheet/iron(loc, 2)
+		new /obj/item/shard(loc)
+		new /obj/item/shard(loc)
+	else
+		new /obj/item/wallframe/status_display(drop_location())
```

Comment:
You need to wrap the NODECONSTRUCT_1 thing in its own check, otherwise this is going to proc for nondeconstructables.

Reply:
Is what I did what you meant?